commons-extension-webapp/src/main/webapp/ckeditor/build-config.js

```diff
@@ -47,69 +49,49 @@ var CKBUILDER_CONFIG = {
 		'gruntfile.js',
 		'less',
 		'node_modules',
+		'package-lock.json',
 		'package.json',
 		'tests'
 	],
 	plugins : {
 		'a11yhelp' : 1,
 		'about' : 1,
 		'basicstyles' : 1,
-		'bidi' : 1,
 		'blockquote' : 1,
 		'clipboard' : 1,
-		'colorbutton' : 1,
-		'colordialog' : 1,
 		'contextmenu' : 1,
-		'copyformatting' : 1,
-		'dialogadvtab' : 1,
-		'div' : 1,
 		'elementspath' : 1,
 		'enterkey' : 1,
 		'entities' : 1,
 		'filebrowser' : 1,
-		'find' : 1,
-		'flash' : 1,
 		'floatingspace' : 1,
-		'font' : 1,
 		'format' : 1,
-		'forms' : 1,
 		'horizontalrule' : 1,
 		'htmlwriter' : 1,
-		'iframe' : 1,
 		'image' : 1,
-		'indentblock' : 1,
 		'indentlist' : 1,
-		'justify' : 1,
-		'language' : 1,
 		'link' : 1,
 		'list' : 1,
-		'liststyle' : 1,
 		'magicline' : 1,
 		'maximize' : 1,
-		'newpage' : 1,
-		'pagebreak' : 1,
+		'pastefromgdocs' : 1,
 		'pastefromword' : 1,
 		'pastetext' : 1,
```

Comment:
won’t this activated unwanted additional plugins?

Reply:
None of the new features will be activated until we change [in our custom config here](https://github.com/Meeds-io/commons/blob/develop/commons-extension-webapp/src/main/webapp/ckeditorCustom/config.js). Therefore, this file is [automatically generated](https://github.com/Meeds-io/commons/blob/b0712baf8c283cb4635cd91d2b5e0620c5d0d200/commons-extension-webapp/src/main/webapp/ckeditor/build-config.js#L7-L8) after building ckeditor and it will not affect the config as mentioned [here](https://github.com/Meeds-io/commons/blob/develop/commons-extension-webapp/src/main/webapp/ckeditor/build-config.js#L22-L24).

Reply:
thanks for the explanation.